src/services/sequelize-service.js

```diff
@@ -36,12 +47,13 @@ async function createSequelizeFile(path) {
  * @param path the path to the new project directory
  * @returns {Promise<void>}
  */
-async function updatePackageJsonWithSequelizeDependency(path) {
+async function updatePackageJsonWithSequelizeDependencies(path) {
   const packageJsonFileName = `${path}/package.json`;
   let projectPackageJson = file.readFile(packageJsonFileName);
   projectPackageJson = JSON.parse(projectPackageJson);
   projectPackageJson.dependencies.sequelize = '^4.33.2';
   projectPackageJson.dependencies.mysql2 = '^1.5.2';
+  projectPackageJson.dependencies.config = '^1.3.0';
```

Comment:
We should make this a part of the general project dependencies. In other words, make it a part of the package.json template.